include/lambdifier/expression.hpp

```diff
@@ -124,6 +140,13 @@ class LAMBDIFIER_DLL_PUBLIC expression
         return p == nullptr ? nullptr : &(p->m_value);
     }
 
+    template <typename T>
```

Comment:
In pagmo we have the same name, ``extract()``, also for the unsafe version. Can you rename this?

Reply:
sure. I am always amazed how the compiler picks up the correct one :)